--- BUILD/pandora_server/util/tentacle_serverd.chkconfig	2010-05-21 00:52:50.000000000 +0900
+++ BUILD/pandora_server/util/tentacle_serverd	2010-07-01 21:43:15.000000000 +0900
@@ -7,6 +7,9 @@
 #
 # System startup script for Tentacle Server
 #
+# chkconfig: - 80 10
+# description: Tentacle Server startup script
+#
 ### BEGIN INIT INFO
 # Provides:       tentacle_server
 # Required-Start: $network
@@ -51,7 +54,7 @@
 
 TENTACLE_ADDR="0.0.0.0"
 TENTACLE_PORT="41121"
-TENTACLE_EXT_OPTS="-i.*\.conf:conf;.*\.md5:md5"
+TENTACLE_EXT_OPTS="-i'.*\.conf:conf;.*\.md5:md5'"
 
 # Set umask to 0002, because group MUST have access to write files to
 # use remote file management on Pandora FMS Enterprise.
@@ -85,7 +88,7 @@
 			rc_exit	
 		fi
 	
-		sudo -u $TENTACLE_USER ${TENTACLE_PATH}$TENTACLE_DAEMON $TENTACLE_OPTS
+		su -l $TENTACLE_USER -s /bin/sh -c "${TENTACLE_PATH}$TENTACLE_DAEMON $TENTACLE_OPTS"
 		sleep 1
 
 		TENTACLE_PID=`get_pid`
